java/README.md

```diff
@@ -82,28 +84,26 @@ $ ./gradlew :client:test
 
 Other useful gradle developer commands:
 * `./gradlew :client:test` to run client unit tests
+* `./gradlew :integTest:test` to run client examples
 * `./gradlew spotlessCheck` to check for codestyle issues
 * `./gradlew spotlessApply` to apply codestyle recommendations
+* `./gradlew :examples:run` to run client examples
 * `./gradlew :benchmarks:run` to run performance benchmarks
 
 ## Basic Examples
 
 ### Standalone Redis:
 
 ```java
-import glide.Client;
-import glide.Client.SingleResponse;
+import glide.api.RedisClient;
 
-Client client = new Client();
+RedisClient client = RedisClient.CreateClient();
 
-SingleResponse connect = client.asyncConnectToRedis("localhost", 6379);
-connect.await().isSuccess();
+CompletableFuture<String> setResponse = client.set("key", "foobar");
+setResponse.get() == "OK";
```

Comment:
this, and line 106, isn't proper java code. please use asserts, prints, or something else which is meaningful.